cuqi/_cuqi_global_vars.py

```diff
@@ -0,0 +1,7 @@
+""" This module defines variables that are used throughout CUQIpy. The values of these variables are not meant to be changed by the user."""
+
+_disable_warning_msg = lambda module_name: "To disable "+\
+"warnings for a given module or library, "+\
+"you can use the method `warnings.filterwarnings`,"+\
+" e.g.: warnings.filterwarnings(action='ignore', "+\
+f"module=r'"+module_name+"')."
```

Comment:
I like the idea of having this area for warning messages etc. I would consider renaming the file to something like cuqi/_messages or something like that. Global variables should likely be in the config file.

Reply:
👍